telemetry/csharp/AwsToolkit.Telemetry.Events.Generator.Tests/DefinitionsBuilderTests.cs

```diff
@@ -7,27 +7,43 @@ namespace Amazon.AwsToolkit.Telemetry.Events.Generator.Tests
     public class DefinitionsBuilderTests
     {
         private readonly DefinitionsBuilder _sut = new DefinitionsBuilder().WithNamespace("Test");
-        
+        private readonly TelemetryDefinitions _definitions = TelemetryDefinitions.Load("test-data/sampleDefinitions.json");
+
+        /// <summary>
+        /// Treat the sample definitions as if they were the central telemetry definitions
+        /// </summary>
         [Fact]
-        public void Build()
+        public void Build_CommonDefinition()
         {
-            LoadDefinitions("sampleDefinitions.json");
-            AssertGeneratedCode("expectedCode.txt");
+            _sut
+                .AddMetrics(_definitions.metrics)
+                .AddMetricsTypes(_definitions.types);
+
+            AssertGeneratedCode("test-data/expectedCode.txt");
         }
 
-        private void LoadDefinitions(string path)
+        /// <summary>
+        /// Treat the sample definitions as if they were the repo-specific "supplemental" telemetry definitions
```

Comment:
Is it customary to comment a C# test like this? In other languages I'd have codified the expected behaviour into the method names `Reference_Definitions_Are_Treated_As_Repo_Specific_Supplementary_Defintions`

Reply:
I'm not aware of a language specific convention. I felt the comments were useful.